### PR TITLE
Remove deprecated config and simplify guides for all devices

### LINKED ISSUE #307: [maint] Fix Nextcloud Guides

### PR DESCRIPTION
**Pull request by kn0wmad**
Closes #307

### CODE REVIEW

File: site/source/user-manual/service-guides/nextcloud/nextcloud-ios.rst [hunk 1]
```diff
@@ -44,12 +35,11 @@ Tor Setup
...
-4. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with http:// and end with .onion. Click Next.
-5. This will launch your browser and prompt you to log in to your account. Log in and then grant access as we did for LAN.
-6. That's it! You may wish to set up some select folders for remote sync, but for large files, it is best to sync on LAN only, so you can "Skip folders configuration" on the resulting screen if you wish. Check your connection by clicking the newly created account in the client app.
+1. On your desktop application, click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
+2. Close the Settings screen and click the account in the top left again, then "Add Account."
+3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with http:// and end with .onion. Click Next.
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-ios.rst:40]: This button actually says "Log in to your Nextcloud" (if that matters).

File: site/source/user-manual/service-guides/nextcloud/nextcloud-android.rst [hunk 2]
```diff
@@ -44,12 +35,11 @@ Tor Setup
...
-4. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with http:// and end with .onion. Click Next.
-5. This will launch your browser and prompt you to log in to your account. Log in and then grant access as we did for LAN.
-6. That's it! You may wish to set up some select folders for remote sync, but for large files, it is best to sync on LAN only, so you can "Skip folders configuration" on the resulting screen if you wish. Check your connection by clicking the newly created account in the client app.
+1. On your desktop application, click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
+2. Close the Settings screen and click the account in the top left again, then "Add Account."
+3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with http:// and end with .onion. Click Next.
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-android.rst:40]: This button actually says "Log in to your Nextcloud" iffin ya care.  I think "Log in" is clear enough though, personally.

File: site/source/user-manual/service-guides/nextcloud/nextcloud-linux.rst [hunk 3]
```diff
@@ -62,12 +53,11 @@ Tor Setup
...
-4. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with http:// and end with .onion. Click Next.
-5. This will launch your browser and prompt you to log in to your account. Log in and then grant access as we did for LAN.
-6. That's it! You may wish to set up some select folders for remote sync, but for large files, it is best to sync on LAN only, so you can "Skip folders configuration" on the resulting screen if you wish. Check your connection by clicking the newly created account in the client app.
+1. On your desktop application. Click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
+2. Close the Settings screen and click the account in the top left again, then "Add Account."
+3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces - Tor. This must start with http:// and end with .onion. Click Next.
```
Comment (gStart9) [inline site/source/user-manual/service-guides/nextcloud/nextcloud-linux.rst:58]: This button actually says "Log in to your Nextcloud" (At least on Debian) iffin ya care. I think "Log in" is clear enough though, personally.

**Review by gStart9 — APPROVED** [review]
Looks good (see comment).

File: site/source/user-manual/service-guides/nextcloud/nextcloud-linux.rst (hunk 3 above)
Comment (kn0wmad) [reply site/source/user-manual/service-guides/nextcloud/nextcloud-linux.rst:58]: I think it's okay, but good eye

**Pull request merged**